raving-remoras/bangazon-site
# Add Product Validation

# Description
Adds validation that will not allow you to add a negative quantity to a product. 

Given a user wants to sell a product
When the user is filling out the product form
And enters is a price that is greater than $10,000
Then the user should be notified that the price must be $10,000 or less

## Number of Fixes
Fixed tests for product category and view products by category

## Related Ticket(s)
#66 #24 

## Steps to Test Solution

1. When logged in, try to add a product with negative quantity, you should recieve an error stating that the quantity must be equal to or greater than 0
1. When logged in, try to add a product with price greater than $10,000, you should recieve an error stating that the price must be less than or equal to 10,000
1. Pyman Test

## Testing


## Documentation
